docs/knowledgebase/getting-started/glossary.md

```diff
@@ -43,59 +43,84 @@ overall amount of votes wins. Notably:
 - voting for all candidates is exactly equivalent to voting for none; and
 - it is possible to vote "against" a single candidate by voting for all other candidates.
 
+Approval voting is used by the [FRAME Collective pallet](../runtime/frame#collective) that
```

Comment:
Collective does not use approval voting at all. Elections Phragmen does and then changes the instantiation of Collective that is the Council using it. Technical Committee is also an instantiation of Collective but does not use approval voting. https://github.com/paritytech/polkadot/blob/6577751/runtime/polkadot/src/lib.rs#L491
```suggestion
Approval voting is used by the [FRAME Elections Phragmen pallet](../runtime/frame#elections-phragmen) that
```